### DISCUSSION TITLE
template syntax support

### DISCUSSION THREAD

**Discussion by badhri17** [withastro/roadmap#737] [Proposal]
### Body

Hello I just started using astro a couple  months ago.
everything feels so intuitive and amazing about this framework except the JSX like syntax that astro is using by default 
a lot of developers with Vue , Svelte, Angular background felling annoyed by the JSX syntax
adding a way to write template syntax in astro will be a game changer feature.

**Comment by ghost** [comment]
This would be sick

**Comment by djmtype** [comment]
I inquired about this years ago, too, on Discord. @natemoo-re responded that it was considered, where the syntax would be more akin to Solid. Example: https://docs.solidjs.com/reference/components/for

Ripple, a new TS UI framework, also uses a similar syntax. 

I hope the Astro team takes the time to reconsider. Nested JSX is just awful to wrangle.
